group.go

```diff
@@ -443,6 +444,79 @@ func (g *group) LocalRemove(key string) {
 	})
 }
 
+func (g *group) RemoveKeys(ctx context.Context, keys ...string) error {
+	if len(keys) == 0 {
+		return nil
+	}
+
+	g.Stats.BatchRemoves.Add(1)
```

Comment:
Let's be consistent with our naming. 
```suggestion
	g.Stats.RemoveKeysRequests.Add(1)
```